hack/dockerized

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+DIR="$(
+    cd "$(dirname "$0")"
+    pwd
+)"
+DOCKER_DIR=${DIR}/docker-builder
+KUBEVIRT_DIR="$(
+    cd "$DIR/../"
+    pwd
+)"
+
+BUILDER=${BUILDER_NAME:-kubevirt${USER_ID}builder}
+
+(cd ${DOCKER_DIR} && docker build . -t ${BUILDER})
+docker volume create --name ${BUILDER}
+docker run -v "${BUILDER}:/root:rw,z" --rm -it ${BUILDER} mkdir -p /root/go/src/kubevirt.io/kubevirt/_out
+
+_rsync() {
+    RSYNC_CID=$(docker run -d -v "${BUILDER}:/root:rw,z" -p 127.0.0.1:10873:873 ${BUILDER} /usr/bin/rsync --no-detach --daemon --verbose)
+    echo "Syncing ..."
+    # TODO improve that
+    sleep 1
```

Comment:
Why the sleep here?

Reply:
I wonder why that ever worked for you without the `sleep`. the rsync daemon is just not ready fast enough, that is the reason why the follow up rsync fails for me if I don't wait.